### ISSUE TITLE
The connect.py needs more precise documentation & fix for region in account url

### ISSUE THREAD

**Issue by shraddha-kr** [team-data-science/snowflake-for-data-engineers#1]
#1 - The script documentation need to mention, how to find a specific user's account url, and guide the user how to locate them on the snowflake platform.  Since the region for every user using this script may vary, and the script may fail if used as is.
#2 - account=f'{snowflake_id}.snowflakecomputing.com', This won't work, as the account url is as below
https://{snowflake_id}.{region}.snowflakecomputing.com

**Comment by team-data-science** [comment]
Didn't I show this in the video what to put in there and how to get it?
I was under the impression that I did

**Comment by shraddha-kr** [comment]
Yes, The next video has it, however, the account url will need the region to be passed.
